environments/eck-ror/start.sh

```diff
@@ -114,7 +141,7 @@ docker exec ror-eck-control-plane bash -c 'cd ror && ls | xargs -n 1 kubectl app
 
 echo ""
 echo "------------------------------------------"
-echo "ECK and ROR is being bootstrapped. Wait for all pods to be run and then open your browser and try to access https://localhost:5601/ (credentials admin:admin)"
+echo "ECK and ROR is being bootstrapped. Wait for all pods to be run and then open your browser and try to access https://localhost:5601/"
```

Comment:
If we don't want to write credentials explicitly, what about instructing a user where they can be found?

Reply:
I'm not sure if we need it in this repo. We start envs with some initial configs, but then they can be overwritten by our tests. 

So, we could write here sth like "hey, check your env's current configuration to find out credentials", but I'm not sure if it changes much the current state. 

WDYT?

Reply:
Ok, I forgot that it's not a ror-sandbox repo but rather our internal project, so we are the users. Yeah, let's leave it as it is